src/pages/contact/Contact.tsx

```diff
@@ -0,0 +1,165 @@
+      ['discord', 'banny#4200'],
+      ['twitter', '@banny'],
+      ['telegram', '@banny'],
+    ])
+
+    setContactPlaceholder(examples.get(option) ?? 'Your contact')
+  }
+
+  const contactTypes = (
+    <Form.Item name="contactPlatform" initialValue="email" className="mb-0">
+      <Select
+        className="min-w-[9em]"
+        onSelect={handleSelect}
+        options={[
+          { value: 'email', label: t`Email` },
+          { value: 'discord', label: t`Discord` },
+          { value: 'twitter', label: t`Twitter` },
+          { value: 'telegram', label: t`Telegram` },
+        ]}
+      />
+    </Form.Item>
+  )
+
+  return (
+    <>
+      <div className="mx-auto max-w-5xl px-10">
+        <Row align="middle" gutter={40}>
+          <Col xs={24} md={14}>
+            <h1 className="m-0 my-4 text-4xl font-semibold">
+              <Trans>Contact</Trans>
+            </h1>
+            <p>
+              <Trans>
+                Message a DAO contributor for feature requests, help planning
+                your project, or to have your questions and inquiries answered.
+              </Trans>
+            </p>
+            <Divider />
+            <Form
+              onFinish={onFormSubmit}
+              form={form}
+              layout="vertical"
+              className="max-w-4xl"
+            >
+              <Form.Item name="name" label={t`Your Name`}>
+                <Input placeholder="Banny the Banana" />
+              </Form.Item>
+              <Form.Item name="contact" label={t`Where to Contact You`}>
+                <Input
+                  addonAfter={contactTypes}
+                  placeholder={contactPlaceholder}
+                />
+              </Form.Item>
+              <Form.Item
+                name="subject"
+                label={t`Subject`}
+                initialValue="project help"
+              >
+                <Select
+                  options={[
+                    {
+                      value: 'project help',
+                      label: t`Get help planning or configuring my project.`,
+                    },
+                    { value: 'feature request', label: t`Request a feature.` },
+                    {
+                      value: 'media inquiry',
+                      label: t`I have a media inquiry.`,
+                    },
+                    { value: 'general question', label: t`I have a question.` },
+                    { value: 'other', label: t`Other.` },
+                  ]}
+                />
+              </Form.Item>
+              <Form.Item name="message" label={t`Your Message`}>
+                <Input.TextArea
+                  maxLength={500}
+                  showCount={true}
+                  onChange={() => setSubmitDisabled(false)}
+                />
+              </Form.Item>
+              <Button
+                size="large"
+                type="primary"
+                htmlType="submit"
+                disabled={submitDisabled}
```

Comment:
**suggestion** if we make the `message` a required field, then we can remove this disabled state and just let the user click `submit` to trigger the validation. Typically a nicer UX, because the user explicitly knows what they need to fix.

Reply:
Sounds good to me! Replacing with antd form `required` logic.